frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -559,11 +559,44 @@ function DashboardInner(props: DashboardProps) {
              * in Redux state which kicks off a fetch for the dashboard cards.
              */}
             <DashboardHeader
-              {...props}
+              dashboardId={dashboardId}
+              isEditing={isEditing}
```

Comment:
`isEmpty`, `isPublic` & `setRefreshElapsedHook` are missing, they're used in `getDashboardActions`

Reply:
CI will (hopefully?) break since these ones don't exist. I've done some work on making `getDashboardActions` explicit as well so that will help in the future

Reply:
CI won't fail since all of these props have been declared as optional

Reply:
actually, isEmpty and isPublic are not available on DashboardInner props, that's why I didn't add them.

thanks for catching `setRefreshElapsedHook`